### ISSUE TITLE
PageScroll plugin does not display correctly (CSS)

### ISSUE THREAD

**Issue by Darknico** [dragomano/Light-Portal#192]
**Describe the bug** (**Опишите баг**)
Plugin PageScroll not work

**To Reproduce** (**Как воспроизвести, по шагам**)
1. Go to plugin
2. Enable PageScroll
3. Not show the lateral button but a strange block in the frontpage

**Expected behavior** (**Ожидаемое поведение — что хотели получить вместо бага**)
Add the button up/down in the lateral

**Screenshots**
![immagine](https://github.com/dragomano/Light-Portal/assets/3503100/a3e463c6-9522-44eb-8f0e-4bda21e24faf)

**Environment**
- Version/Git revision: 2.5.2
- Database Type: MySQL
- Database Version: 5.7.28
- PHP Version: 8.1.1
- SMF Version: 2.1.4

**plugin**
- Name: PageScroll
- Version: 17.01.24

**Comment by Darknico** [comment]
It isn't a block, the button is inside the frontage and if click move down =D

**Comment by dragomano** [comment]
Did you clean cache?

**Comment by Darknico** [comment]
Yes, cleaned.
I tested this plugin in another forum and same issue

**Comment by Darknico** [comment]
>Is there Themes/default/css/light_portal/plugins.css?
>Is there Sources/LightPortal/Addons/PageScroll/style.css?

yes
And it is strange, in plugins.css there are the rules

>Try to put content from the plugin's style.css to the end of your index.css and clean cache.

If add manually the css yes, work.
But it is not correct add manually :)

**Comment by dragomano** [comment]
Yes, when you enable/disable each plugin that has styles, the plugins.css file should be updated, check this.

**Comment by Darknico** [comment]
WTF? I removed the rules added manually and now work? SMF recreate the minified files?

**Comment by dragomano** [comment]
Try to disable minify of CSS/JS and check whether plugins.css is added to the list of files to be loaded.

**Comment by Darknico** [comment]
Another test:
if disable the plugin, don't remove the rules in plugins.css, it is normal?

**Comment by Darknico** [comment]
> Try to disable minify of CSS/JS and check whether plugins.css is added to the list of files to be loaded.
yes, it is added
![immagine](https://github.com/dragomano/Light-Portal/assets/3503100/ef6fc6d0-a295-41b1-bf94-6f0e69b8b6b5)

**Comment by dragomano** [comment]
> Another test: if disable the plugin, don't remove the rules in plugins.css, it is normal?
Probably not. :)

Now remove all files like minified_*.css in your Themes/default/css dir and enable minify JS/CSS option again.

**Comment by Darknico** [comment]
done

Work correctly but it is strange...

**Comment by dragomano** [comment]
Yes, for the moment they remain. I'll think about how to remove them when disabling the plugin.

**Comment by Darknico** [comment]
I can't understand what could have happened.

For test, I disable page scroll plugin and remove manually the rules in plugin.css.
Enable the plugin, update plugin.css with the rules

Work correctly, good, but why before not?
I don't think I'll ever find an answer :)

> Yes, for the moment they remain. I'll think about how to remove them when disabling the plugin.

Ok, thanks!

**Comment by dragomano** [comment]
I was finally able to reproduce the problem mentioned, but only after I deleted the plugins.css file, and then enabled the PageScroll plugin. Then I turned it off and on again and the problem disappeared. It looks like the first version of the plugins.css file (without PageScroll styles) was attempted first, and the second time the new version (with PageScroll styles) was already loaded. The wonders of technology.

In the current version of the portal, after enabling any plugin that contains a stlye.css file, the contents of that file will included into plugins.css, which is then minified. After disabling the plugin, it is impossible to remove its styles from the file, because there is simply no necessary method for this in the `MatthiasMullie\Minify\Minify` class. Alternatively, we could just delete the plugins.css file after enabling/disabling each plugin, but I decided not to stress hard disks unnecessarily. In any case, plugins.css is still minified.
